src/storage/mongodb/TenantStorage.ts

```diff
@@ -190,10 +190,8 @@ export default class TenantStorage {
         $addFields: {
           logo: {
             $concat: [
-              `${Utils.buildRestServerURL()}/client/util/TenantLogo?ID=`,
-              { $toString: '$_id' },
-              '&LastChangedOn=',
-              { $toString: '$lastChangedOn' }
+              `${Utils.buildRestServerURL()}/v1/util/tenants/logo?ID=`,
+              { $toString: '$_id' }
```

Comment:
The last change timestamp is necessary to force the browser to reload the cache when there is change in the logo ;-).

Reply:
Ok ! Fixed to concatenation only if possible